doc/hotspot-style.md

```diff
@@ -662,13 +666,17 @@ references to non-function-local `thread_local` variables defined in a
 different translation unit, even if they don't need dynamic
 initialization.  Dynamic initialization and destruction of
 namespace-scoped thread local variables also has the same ordering
-problems as for ordinary namespace-scoped variables.
+problems as for ordinary namespace-scoped variables. So we avoid use of
```

Comment:
Should be s/namespace-scoped/non-local/ (two places).  This was overlooked by JDK-8272691.  (Mea culpa.) And yes, "non-local thread local"  (or "non-local thread-local") looks a little weird.  The standard (almost) doesn't use "thread-local" as a term, instead talking about variables with "thread storage duration".

Reply:
Fixed. And I changed "thread local" to "`thread_local` for consistency and avoid the weirdness.